loadbalancer/src/main/java/com/io/loadbalancer/exceptions/NoIMInstanceAvailableException.java

```diff
@@ -0,0 +1,20 @@
+package com.io.loadbalancer.exceptions;
+
+public class NoIMInstanceAvailableException extends Exception{
+
+    public NoIMInstanceAvailableException() {
+        super();
+    }
+
+    public NoIMInstanceAvailableException(String message, Throwable cause) {
```

Comment:
do you have to specify those methods though? `public class ... extends Exception {}` doesn't work?

Reply:
Yes, that's enough. Done